packet/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -1,118 +1,120 @@
 
   # DNS Zone where record should be created
-  zone_id = "${var.dns_zone_id}"
+  zone_id = var.dns_zone_id
 
-  name = "${format("%s-etcd%d.%s.", var.cluster_name, count.index, var.dns_zone)}"
+  name = format("%s-etcd%d.%s.", var.cluster_name, count.index, var.dns_zone)
   type = "A"
   ttl  = 300
 
   # private IPv4 address for etcd
-  records = ["${element(packet_device.controllers.*.access_private_ipv4, count.index)}"]
+  records = [packet_device.controllers[count.index].access_private_ipv4]
 }
 
 # DNS record for the API servers
 resource "aws_route53_record" "apiservers" {
-  zone_id = "${var.dns_zone_id}"
+  zone_id = var.dns_zone_id
 
-  name = "${format("%s.%s.", var.cluster_name, var.dns_zone)}"
+  name = format("%s.%s.", var.cluster_name, var.dns_zone)
   type = "A"
   ttl  = "300"
 
   # TODO - verify that a multi-controller setup actually works
-  records = ["${packet_device.controllers.*.access_public_ipv4}"]
+  records = packet_device.controllers.*.access_public_ipv4
 }
 
 resource "aws_route53_record" "apiservers_private" {
-  zone_id = "${var.dns_zone_id}"
+  zone_id = var.dns_zone_id
 
-  name = "${format("%s-private.%s.", var.cluster_name, var.dns_zone)}"
+  name = format("%s-private.%s.", var.cluster_name, var.dns_zone)
   type = "A"
   ttl  = "300"
 
   # TODO - verify that a multi-controller setup actually works
-  records = ["${packet_device.controllers.*.access_private_ipv4}"]
+  records = packet_device.controllers.*.access_private_ipv4
 }
 
 resource "packet_device" "controllers" {
-  count            = "${var.controller_count}"
+  count            = var.controller_count
   hostname         = "${var.cluster_name}-controller-${count.index}"
-  plan             = "${var.controller_type}"
-  facilities       = ["${var.facility}"]
-  operating_system = "${var.ipxe_script_url != "" ? "custom_ipxe" : format("flatcar_%s", var.os_channel)}"
+  plan             = var.controller_type
+  facilities       = [var.facility]
+  operating_system = var.ipxe_script_url != "" ? "custom_ipxe" : format("flatcar_%s", var.os_channel)
   billing_cycle    = "hourly"
-  project_id       = "${var.project_id}"
-  user_data        = "${var.ipxe_script_url != "" ? element(data.ct_config.controller-install-ignitions.*.rendered, count.index) : element(data.ct_config.controller-ignitions.*.rendered, count.index)}"
+  project_id       = var.project_id
+  user_data        = var.ipxe_script_url != "" ? data.ct_config.controller-install-ignitions[count.index].rendered : data.ct_config.controller-ignitions[count.index].rendered
 
   # If not present in the map, it uses ${var.reservation_ids_default}.
-  hardware_reservation_id = "${lookup(var.reservation_ids, format("controller-%v", count.index), var.reservation_ids_default)}"
-
-  ipxe_script_url  = "${var.ipxe_script_url}"
-  always_pxe       = "false"
+  hardware_reservation_id = lookup(
+    var.reservation_ids,
+    format("controller-%v", count.index),
+    var.reservation_ids_default,
+  )
```

Comment:
is this new style generated by `terraform fmt` or the upgrade script?

Reply:
It's done by the `terraform 0.16upgrade`

Reply:
0.12 I guess? Thanks!

Reply:
I'm already afraid of `0.16upgrade` :joy:

Reply:
Not as much as I am. :rofl: